src/generators/net_caffe_recurrent.h

```diff
@@ -0,0 +1,83 @@
+/**
+ * DeepDetect
+ * Copyright (c) 2014-2016 Emmanuel Benazera
+ * Author: Emmanuel Benazera <beniz@droidnik.fr>
```

Comment:
I might go see my doctor and skip work today, I can't remember writing this piece!